src/frontend/src/pages/Calendar/CalendarComponents/CalendarDayCard.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Card, CardContent, Grid, IconButton, Typography } from '@mui/material';
+import AddCircleOutlineIcon from '@mui/icons-material/AddCircleOutline';
+import { DesignReview } from 'shared';
+import { NOON_IN_MINUTES } from '../../../utils/design-review.utils';
+
+interface CalendarDayCardProps {
+  myDate: Date;
+  events: DesignReview[];
+}
+
+const CalendarDayCard: React.FC<CalendarDayCardProps> = ({ myDate, events }) => {
+  const DayCardTitle = () => (
+    <Grid container alignItems="center" margin={0} padding={0}>
+      <Grid item>
+        <IconButton>
+          <AddCircleOutlineIcon fontSize="small" />
+        </IconButton>
+      </Grid>
+      <Grid item xs display="flex" justifyContent="flex-end">
+        <Typography variant="h6" marginRight={1}>
+          {myDate.getDate()}
+        </Typography>
+      </Grid>
+    </Grid>
+  );
+
+  const EventCard = (event: DesignReview) => {
+    const name = event.designReviewId;
+    // number of minutes after midnight
+    const time = event.meetingTimes[0] * 15 + 9 * 60;
+    const minutes = time % 60;
+    const hours = ((time - minutes) / 60) % 12 === 0 ? 12 : ((time - minutes) / 60) % 12;
+    return (
+      <Box marginLeft={0.5}>
+        <Card sx={{ backgroundColor: 'red', borderRadius: 1, minWidth: 140, maxWidth: 140, minHeight: 20, maxHeight: 20 }}>
+          <Typography marginLeft={0.5} marginBottom={0.3}>
+            {name + ' ' + hours + (minutes !== 0 ? ':' + minutes : '') + (time >= NOON_IN_MINUTES ? 'pm' : 'am')}
```

Comment:
I'm thinking it might be best to even have this as a pipe so that we don't have to have all of the calculations and stuff in the card